doc/architecture.rst

```diff
@@ -375,22 +375,25 @@ tickets and session keys.
 
 Smart Daemons Enable Hyperscale
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-In many clustered architectures, the primary purpose of cluster membership is 
-so that a centralized interface knows which nodes it can access. Then the
-centralized interface provides services to the client through a double
-dispatch--which is a **huge** bottleneck at the petabyte-to-exabyte scale.
-
-Ceph eliminates the bottleneck: Ceph's OSD Daemons AND Ceph Clients are cluster
-aware. Like Ceph clients, each Ceph OSD Daemon knows about other Ceph OSD
-Daemons in the cluster.  This enables Ceph OSD Daemons to interact directly with
-other Ceph OSD Daemons and Ceph Monitors. Additionally, it enables Ceph Clients
-to interact directly with Ceph OSD Daemons.
-
-The ability of Ceph Clients, Ceph Monitors and Ceph OSD Daemons to interact with
-each other means that Ceph OSD Daemons can utilize the CPU and RAM of the Ceph
-nodes to easily perform tasks that would bog down a centralized server. The
-ability to leverage this computing power leads to several major benefits:
+A feature of many storage clusters is a centralized interface that keeps track
+of the nodes that the clients are permitted to access. Such centralized
+architectures provide services to clients by means of a double dispatch. At the
+petabyte-to-exabyte scale, such double dispatches generate a significant
+bottleneck.
+
+Ceph eliminates this bottleneck: Ceph's OSD Daemons AND Ceph clients are
+cluster aware. Like Ceph clients, each Ceph OSD Daemon is aware of other Ceph
+OSD Daemons in the cluster. This enables Ceph OSD Daemons to interact directly
+with other Ceph OSD Daemons and to interact directly with Ceph Monitors.  Being
+cluster-aware makes it possible for Ceph clients to interact directly with Ceph
+OSD Daemons.
+
+Because Ceph clients, Ceph monitors, and Ceph OSD daemons can interact with one
```

Comment:
s/can//

This convergence on repeated _Ceph xxx_ is a bit awkward but I can understand the desire for uniform nomenclature.

Reply:
"can" was a just stopgap measure that helped me with the cognitive load of recasting of this paragraph, and you're right. There's no reason to be modal here. Plain old indicative is better.

Reply:
And I envision the reader of the architecture document to be someone who has just finished reading the not-yet-available Beginner's Guide, and who isn't completely sure if a monitor is a Ceph monitor and so on, and so needs to be reminded.

Reply:
Ack.  Plus it's very common for someone to use a search engine to find our works without starting at the beginning.

There was this lawyer on a plane once telling his daughter over the phone about how he bought the regular size candy bar then bought the large size candy bar as well as the regular size candy bar.  Stuff like that I can't unhear.